data/categories.txt

```diff
@@ -1717,19 +1735,20 @@ eu:4Barazaina
 sv:Grönsakshandlare
 th:ร้านขายผัด
 tr:Manav
-uk:Магазин овочів|3овочі|3фрукти
+uk:3овочі|овочів|3фрукти
 vi:Cửa hàng rau củ
 zh-Hans:蔬果零售店|商店
 zh-Hant:蔬果零售店|1購物|店鋪|商店|雜貨店|便利商店
 el:Μανάβικο|οπωροπωλείο|παντοπωλείο|κατάστημα
 sk:Zelovoc
 fa:میوه فروشی
 
-shop-sports|@shopping
+shop-sports|@shopping|@shop
 en:4Sports Goods|U+1F3BF|U+1F3A3|U+1F3C2|U+1F6B4|U+26BD|U+1F3C0|U+1F3C8|U+26BE|U+1F3BE|U+1F3C9|U+26F3
-ru:Магазин спорттоваров|4Спорттовары|товары для спорта
-bg:4Спортни стоки|спортен магазин
+ru:4Спорттовары|Спортивный|Товары для спорта
 ar:أدوات رياضية|رياضة|متجر
+be:4Спартыўны
+bg:4Спортни стоки|спортен магазин
 cs:4Sportovní zboží
 da:4Sportsudstyr|sport
 nl:4Sportartikelen
```

Comment:
nl:Sport

Reply:
Just sport? Will it be ok for shop?

Reply:
No, because Dutch uses mostly no spaces.

```suggestion
nl:4Sportartikelen|sportwinkel
```